tests/functional/u/unused/unused_name_in_string_literal_type_annotation_py39.py

```diff
@@ -0,0 +1,10 @@
+# pylint: disable=missing-docstring
```

Comment:
Why is this test python 3.9 only ?

Reply:
graphlib was introduced in python 3.9.

I added an explicit test for it, because mypy expects TopologicalSorter to have a subscript even though it's not subscriptable at runtime. The only way to properly annotate it at the moment is to use a string.